src/classes/BDI_Donations.cls

```diff
@@ -433,25 +437,29 @@ public class BDI_Donations {
         }
 
         // now copy over all the data to the Opp
-        for (string strDIField : mapDIFieldToOppField.keySet()) {
+        for (String dataImportField : mapDIFieldToOppField.keySet()) {
+
+            String donationRecordTypeName = UTIL_Namespace.StrTokenNSPrefix('Donation_Record_Type_Name__c').toLowercase();
```

Comment:
We should be able to move these variables outside of the for loop, right?

Reply:
you are right.